examples/traffic-light.manifold

```diff
@@ -0,0 +1,36 @@
+* = import Manifold.core
+* = import Manifold.digital
+
+Type TrafficLight = (
+	green:Boolean  = false,
+	yellow:Boolean = false,
+	red:Boolean    = false
+)
+
+GREEN_SECONDS  = 10
+YELLOW_SECONDS = 3
+TOTAL_SECONDS  = 2 * (GREEN_SECONDS + YELLOW_SECONDS)
+
+seconds = Manifold.Digital.count(hz: 1, mod: TOTAL_SECONDS)
+
+if (seconds < GREEN_SECONDS) {
+  north_light = south_light = (green: true)
+  east_light = west_light = (red: true)
+
+} else if (seconds < GREEN_SECONDS + YELLOW_SECONDS) {
+  north_light = south_light = (yellow: true)
+  east_light = west_light = (red: true)
+
+} else if (seconds < GREEN_SECONDS + YELLOW_SECONDS + GREEN_SECONDS) {
+  north_light = south_light = (red: true)
+  east_light = west_light = (green: true)
+
+} else {
+  north_light = south_light = (red: true)
+  east_light = west_light = (yellow: true)
+}
+
+Manifold.Digital.led[0..3]  = TrafficLight north_light
```

Comment:
In some (most?) languages, the `..` operator indicates the end index is exclusive and the `...` operator indicates the end index is inclusive.